taipy/core/reason/reason.py

```diff
@@ -155,3 +155,76 @@ class InvalidUploadFile(Reason, _DataNodeReasonMixin):
     def __init__(self, file_name: str, datanode_id: str):
         Reason.__init__(self, f'The uploaded file {file_name} has invalid data for data node "{datanode_id}"')
         _DataNodeReasonMixin.__init__(self, datanode_id)
+
+
+class EntityDoesNotExist(Reason, _DataNodeReasonMixin):
+    """
+    The entity id provided does not exist in the repository.
+
+    Attributes:
+        entity_id (str): The entity identifier.
+    """
+
+    def __init__(self, entity_id: str):
+        Reason.__init__(self, f"Entity {entity_id} does not exist in the repository.")
+
+
+class JobIsNotFinished(Reason, _DataNodeReasonMixin):
```

Comment:
This Reason is not related to data nodes. It should not inherited from `_DataNodeReasonMixin`.